docs/getting-started/install-configure.md

```diff
@@ -250,7 +248,7 @@ spec:
 Then apply it:
 
 ```console
-kubectl apply -f provider.yaml
+kubectl apply -f providerconfig.yaml
```

Comment:
Can this one also be a snippet URL?

Reply:
GCP `ProviderConfig` object has `spec.projectId` field which user has to edit before applying. It'd have to be something like `curl <snippet> | sed <change things> | kubectl apply -f -` so I just kept it as is because it looked simpler to understand.

Reply:
I wonder if we could just set the env var then do something like we do here: https://github.com/crossplane/provider-gcp/blob/d97211eb06898c04cdc8b1f8af9be5a9bab0ade5/cluster/local/integration_tests.sh#L273

If not, this seems fine to me :+1: